lib/OpenQA/Worker.pm

```diff
@@ -104,6 +107,27 @@ sub catch_exit {
     }
 }
 
+sub setup_logging {
+    my ($app) = @_;
+
+    my $logfile = $ENV{OPENQA_WORKER_LOGFILE} || $worker_settings->{LOG};
+
+    if ($logfile) {
+        $app->log->path("$logfile.$instance");    # So each worker get it's own log
+        $app->log->format(
+            sub {
+                my ($time, $level, @lines) = @_;
+                return '[' . localtime($time) . "] [" . $app->log_name . ":$level] " . join "\n", @lines, '';
+            });
+    }
+    if ($worker_settings->{LOG_LEVEL}) {
```

Comment:
and that as:
```perl
$app->log->level($worker_settings->{LOG_LEVEL}) if $worker_settings->{LOG_LEVEL};
```

but if we want a default except than info, and do not leave that choice to mojo, i'd suggest 
```perl
$app->log->level($worker_settings->{LOG_LEVEL}? $worker_settings->{LOG_LEVEL} : "default");
```